### PR TITLE
[Jupyter] Basic PPS UI

### PR DESCRIPTION
**Pull request by bbonenfant**
This is a very simple (first draft) of the UI for the PPS plugin for Jupyterlab. This doesn't include any backend calls. This my first PR for front end work so I'm sure a lot of it is incorrect, please tear it apart!

Question for @smalyala : I'm not sure that we want this merged into master yet (I don't know how the plugin releases are currently being done). Do you have thoughts about where this line of work should live?

Jira: [INT-665](https://pachyderm.atlassian.net/browse/INT-665)

[INT-665]: https://pachyderm.atlassian.net/browse/INT-665?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by smalyala** [comment]
Let's start a feature branch for the pipeline extension, which we merge into master once the pipeline extension is ready.

File: jupyter-extension/src/plugins/mount/mount.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/mount.tsx:447]: Add this to `setShowFullPageError` below as well

File: jupyter-extension/src/plugins/mount/mount.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/mount.tsx:220]: We touched upon this during the FE walkthrough with @pleeko in Miami, but we probably don't need to use signals here since the pipeline component doesn't have any properties/data that changes like the other components does. Signals are useful in the case where we see that some data has changed so we re-render the components that depend on that data. But since the pipeline component doesn't take any input that could change, we can remove signals.

Now, how this would look with no signals, I defer to Peter.

File: jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx:19]: Probably don't need `showPipeline` here

File: jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx:15]: There's a trailing double quotes here

File: jupyter-extension/src/plugins/mount/mount.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/mount.tsx:422]: We don't need the file browser in the bottom of the extension when we're in pipeline mode right? If not, we can hide it to give more space to the pipeline UI.

File: jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx:161]: Are we not going to add the image and command to the pipeline spec preview?

File: jupyter-extension/src/plugins/mount/components/Pipeline/__tests__/Pipeline.test.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/components/Pipeline/__tests__/Pipeline.test.tsx:19]: You can remove these lines related to `mockIntersectionObserver`. This is used for tests that determine whether an HTML element is visible on the screen (different from when it's rendered but hidden behind another element).

File: jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx
Comment (smalyala) [inline jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx:17]: You can remove `showPipeline` as a prop since it's not used anywhere

File: jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx
Comment (bbonenfant) [reply jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx:161]: I don't think we need to show the user the command, since it will not be useful to them, but I'll add the image here.

File: jupyter-extension/src/plugins/mount/mount.tsx
Comment (bbonenfant) [reply jupyter-extension/src/plugins/mount/mount.tsx:220]: I removed the signal in a way that made sense to me, but I'll wait for Peter's feedback

File: jupyter-extension/src/plugins/mount/mount.tsx
Comment (bbonenfant) [reply jupyter-extension/src/plugins/mount/mount.tsx]: Nice I was wondering how to get rid of that, haha

**Review by pleeko — APPROVED** [review]
LGTM

**Pull request merged**
